### ISSUE TITLE
Missing alt text on imgLarge

### ISSUE THREAD

**Issue by readiescards** [morganestes/h5-lightbox#3]
To make the lightbox more Pinterest friendly I suggest the following modification:

         return this.each(function()
        {

            var $this = $(this),
                imgLarge = $this.attr(opt.target),
                altText = $this.attr("alt"),
                /**
                 * check if the image has the proper data attribute
                 * @param {String} img_large Attribute set in options.
                 * @returns {Boolean}
                 */
                isLightbox = function(imgLarge)
                {
                    return (typeof imgLarge !== "undefined");
                };

            if (isLightbox(imgLarge))
            {
                $this.wrap("<a class='" + opt.wrapperClass + "' href='" + imgLarge + "'" + "' alt='" + altText + "'></a>");
            }

            // lightbox implementation
            $("a." + opt.wrapperClass).click(function(e)
            {
                e.preventDefault();
                var lightbox,
                    lbImageHref = $(this).attr("href"),
                    lbImageAlt = $(this).attr("alt");

                if ($("#lightbox").length)
                {
                    $("#lightbox-img").html("<img src='" + lbImageHref + "'' alt='" + lbImageAlt + "' />");
                    $("#lightbox").show();
                }
                else
                {
                    lightbox =
                        "<div id='lightbox'>" +
                        "<p>Click to close</p>" +
                        "<div id='lightbox-img'>" +
                        "<img src='" + lbImageHref + "'' alt='" + lbImageAlt + "' />" +
                        "</div>" +
                        "</div>";

                    $("body").append(lightbox);
                }
            });

            $(document).on("click", "#lightbox", function()
            {
                $(this).hide();
            });

**Comment by morganestes** [comment]
I appreciate the PR and the suggestion, but I'd prefer to leave out markup that's specific to a particular third-party site. I am interested in figuring out ways to extend it to include sharing, but am planning to leave them out of the core plugin for now.
